WordPress/UITestsFoundation/Screens/Login/QuickStartCustomizeScreen.swift

```diff
@@ -0,0 +1,60 @@
+import ScreenObject
+import XCTest
+
+public class QuickStartCustomizeScreen: ScreenObject {
+
+    private let customizeSiteHeaderGetter: (XCUIApplication) -> XCUIElement = {
+        $0.staticTexts["Customize Your Site"]
+    }
+
+    private let createSiteLabelGetter: (XCUIApplication) -> XCUIElement = {
+        $0.staticTexts["Create your site"]
+    }
+
+    private let checkSiteTitleLabelGetter: (XCUIApplication) -> XCUIElement = {
+        $0.staticTexts["Check your site title"]
+    }
+
+    private let chooseSiteIconLabelGetter: (XCUIApplication) -> XCUIElement = {
+        $0.staticTexts["Choose a unique site icon"]
+    }
+
+    private let reviewSitePagesLabelGetter: (XCUIApplication) -> XCUIElement = {
+        $0.staticTexts["Review site pages"]
+    }
+
+    private let viewSiteLabelGetter: (XCUIApplication) -> XCUIElement = {
+        $0.staticTexts["View your site"]
+    }
+
+    var checkSiteTitleLabel: XCUIElement { checkSiteTitleLabelGetter(app) }
+    var chooseSiteIconLabel: XCUIElement { chooseSiteIconLabelGetter(app) }
+    var createSiteLabel: XCUIElement { createSiteLabelGetter(app) }
+    var customizeSiteHeader: XCUIElement { customizeSiteHeaderGetter(app) }
+    var reviewSitePagesLabel: XCUIElement { reviewSitePagesLabelGetter(app) }
+    var viewSiteLabel: XCUIElement { viewSiteLabelGetter(app) }
+
+    public init(app: XCUIApplication = XCUIApplication()) throws {
+        try super.init(
+            expectedElementGetters: [ customizeSiteHeaderGetter ],
+            app: app
+        )
+    }
+
+    public func verifyCustomizeSiteListDisplayed() -> Self {
```

Comment:
Nitpick: It's useful to inject the `#file` and `#line` when writing assertions outside a test case, so that the test failure reported will identify them in the caller's location.

Example: https://github.com/wordpress-mobile/WordPress-iOS/blob/591d8684a998f16227d133761203dde8071c6363/WordPress/UITestsFoundation/Globals.swift#L106-L107

Reply:
good point, updated in https://github.com/wordpress-mobile/WordPress-iOS/pull/21194/commits/7b67b08dfa6d5ddbfabdb53d1f3ee02895dab210 for all the new assertions